docs/integrations/builtin/app-nodes/n8n-nodes-base.googlesheets/sheet-operations.md

```diff
@@ -0,0 +1,220 @@
+
+Enter these parameters:
+
+- **Credential to connect with**: Create or select an existing [Google Sheets credentials](/integrations/builtin/credentials/google/).
+- **Resource**: Select **Sheet Within Document**.
+- **Operation**: Select **Get Row(s)**.
+- **Document**: Choose a spreadsheet that contains the sheet you want to get rows from. 
+    - Select **From list** to choose the spreadsheet title from the dropdown list, **By URL** to enter the url of the spreadsheet, or **By ID** to enter the `spreadsheetId`. 
+    - You can find the `spreadsheetId` in a Google Sheets URL: `https://docs.google.com/spreadsheets/d/spreadsheetId/edit#gid=0`. 
+- **Sheet**: Choose a sheet you want to read rows from.
+    - Select **From list** to choose the sheet title from the dropdown list, **By URL** to enter the url of the sheet, **By ID** to enter the `sheetId`, or **By Name** to enter the name of the sheet.
+    - You can find the `sheetId` in a Google Sheets URL: `https://docs.google.com/spreadsheets/d/aBC-123_xYz/edit#gid=sheetId`.
+- **Filters**: Set filters to return a limited set of results: By default, the node returns all rows in the sheet.
+  	- **Column**: Select the column in your sheet to search against.
+  	- **Value**: Enter a cell value to search for. You can drag input data parameters here.
+	If your filter matches multiple rows, n8n returns the first result. If you want all matching rows: 
+    	1. Under **Options**, select **Add Option** > **When Filter Has Multiple Matches**.
+        2. Change **When Filter Has Multiple Matches** to **Return All Matches**.
+
+### Options
+
+- **Data Location on Sheet**: Use this option to specify a data range. By default, n8n will detect the range automatically until the last row in the sheet. 
+- **Output Formatting**: Use this option to choose how n8n formats the data returned by Google Sheets.
+  - **General Formatting**: 
+    - **Values (unformatted)** (default): n8n removes currency signs and other special formatting. Data type remains as number.
+    - **Values (formatted)**: n8n displays the values as they appear in Google Sheets (for example, retaining commas or currency signs) by converting the data type from number to string.
+    - **Formulas**: n8n returns the formula. It doesn't calculate the formula output. For example, if a cell B2 has the formula `=A2`, n8n returns B2's value as `=A2` (in text). Refer to [About date & time values | Google Sheets](https://developers.google.com/sheets/api/guides/formats#about_date_time_values){:target=_blank .external-link} for more information. 
+  - **Date Formatting**: Refer to [DateTimeRenderOption | Google Sheets](https://developers.google.com/sheets/api/reference/rest/v4/DateTimeRenderOption){:target=_blank .external-link} for more information.   	- **Formatted Text** (default): As displayed in Google Sheets, which depends on the spreadsheet locale. For example `01/01/2024`.  	- **Serial Number**: Number of days since  December 30th 1899. 
```

Comment:
```suggestion
  - **Date Formatting**: Refer to [DateTimeRenderOption | Google Sheets](https://developers.google.com/sheets/api/reference/rest/v4/DateTimeRenderOption){:target=_blank .external-link} for more information.
     	- **Formatted Text** (default): As displayed in Google Sheets, which depends on the spreadsheet locale. For example `01/01/2024`.
     	- **Serial Number**: Number of days since  December 30th 1899. 
```